app/Http/Controllers/ClassController.php

```diff
@@ -4,12 +4,23 @@
 
 use App\Models\Classe;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
```

Comment:
do you really need this Facade?

Reply:
I was trying to remove the old image using it but i couldnt do it.